Meshtastic/Helpers/BLEManager.swift

```diff
@@ -835,8 +835,8 @@ class BLEManager: NSObject, CBPeripheralDelegate, MqttClientProxyManagerDelegate
 					let traceRoute = getTraceRoute(id: Int64(decodedInfo.packet.decoded.requestID), context: context)
 					traceRoute?.response = true
 					if routingMessage.route.count == 0 {
-						let snr = routingMessage.snrBack.count > 0 ? routingMessage.snrBack[0] / 4 : 0
-						traceRoute?.snr = Float(snr)
+						let snr = routingMessage.snrBack.count > 0 ? Float(routingMessage.snrBack[0]) / 4 : 0.0
```

Comment:
zero is a bad default value for these unless it's impossible for SNR reading values to have that. It seems like it's possible tho since it frequently ranges from -20 to +10.

Reply:
Agreed. Let me rework these spots a bit.

Reply:
The firmware will set unknown/unset snr values in routing messages to -128. When we divide by 4 that becomes -32 which is outside the range of valid snr values.
This code is now setting the snr as unknown when it sees -32.